discussions/views_test.py

```diff
@@ -186,11 +188,23 @@ def test_create_channel_user_without_permission(patched_users_api):
     assert resp.status_code == 403
```

Comment:
Can you add a check that `add_channel` is also never called, just to verify that the error code and behavior match up?

Reply:
sure